frontend/public/components/pod.tsx

```diff
@@ -556,7 +556,7 @@ const PodMetrics: React.FC<PodMetricsProps> = ({ obj }) => {
         <GridItem xl={6} lg={12}>
           <Card className="resource-metrics-dashboard__card">
             <CardHeader>
-              <CardTitle>{t('public~CPU usage')}</CardTitle>
+              <CardTitle>{t('public~CPU usage(millicores)')}</CardTitle>
```

Comment:
```suggestion
              <CardTitle>{t('public~CPU usage (millicores)')}</CardTitle>
```

Reply:
Do we know if this is always really millicores? If the pod is using more than one core, wouldn't the chart ticks adjust to show cores instead of millicores?

Reply:
Right, the comments in [CONSOLE-1567](https://issues.redhat.com/browse/CONSOLE-1567) seem to be related to this.

Reply:
Right, if the CPU usage is high, then it will not be millicores.